libs/storybook-abs-paths/src/index.ts

```diff
@@ -0,0 +1,29 @@
+import { existsSync } from 'fs';
+import { sep } from 'path';
+import type { Compiler } from 'webpack';
+
+export class AbsolutePathRemapper {
```

Comment:
webpack plugin to support absolute paths in storybook monorepo.

I should really publish this...